lib/Importer.js

```diff
@@ -154,24 +154,32 @@ class Importer {
     const unusedImportVariables = new Set();
     Object.keys(unusedVariables).forEach((variable) => {
       const lineNumbers = unusedVariables[variable];
-      if (anyNumbersWithinRange(lineNumbers, oldImports.range)) {
+      if (oldImports.range.length > 1 &&
```

Comment:
I read this post about not using `>` and thought it was an interesting idea. http://llewellynfalco.blogspot.com/2016/02/dont-use-greater-than-sign-in.html